### PR TITLE
merging mq demo app

### PR DESCRIPTION
**Pull request by rin0o1**

### CODE REVIEW

**Review by chughts — CHANGES_REQUESTED** [review]
Please review and accept the CLA. 

Some files can be removed from the pull request.

File: ibm-messaging-mq-cloud-showcase-app/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/CONTRIBUTING.md:1]: Contributing file should be at mq-dev-patterns root.

File: ibm-messaging-mq-cloud-showcase-app/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/LICENSE:1]: This a duplicate of the license at the root of the dev-patterns repository. Can be removed from here.

File: ibm-messaging-mq-cloud-showcase-app/QMGR-docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+services:
+  mq:
+    image: "icr.io/ibm-messaging/mq:latest"
+    ports:
+      - 9443:9443
+      - 1414:1414
+    environment:
+      - LICENSE=accept
+      - MQ_QMGR_NAME=QM1
+      - MQ_APP_PASSWORD=${APP_PASSWORD}
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/QMGR-docker-compose.yml:10]: use the syntax ${APP_PASSWORD:?err} to signal failure when password has not been provided.

File: ibm-messaging-mq-cloud-showcase-app/QMGR-docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+services:
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/QMGR-docker-compose.yml:1]: Only include the yml files that the tutorial or the readme refer to

File: ibm-messaging-mq-cloud-showcase-app/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+IBM MQ MESSAGING PLAYGROUND APPLICATION
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/README.md:1]: Add a link to the tutorial (or a placeholder) to remind you to.

File: ibm-messaging-mq-cloud-showcase-app/backend/AWS-showcase-backend/keys/clientkey.sth [hunk 6]
```diff
@@ -0,0 +1,2 @@
+n��P��]������}�F�K��ȭq��K30Pu�dl4���%a�����P���k�rz�N9�	&�����Ц�E&�z	?8��V,�p=�rN����<&�SP�eӫ}�e&�q�
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/backend/AWS-showcase-backend/keys/clientkey.sth:1]: these files should be removed.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.circleci/config.yml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+version: 2
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.circleci/config.yml:1]: These look like editor configurations that shouldn't be included in this pull request.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.editorconfig [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# top-most EditorConfig file
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.editorconfig:1]: unneeded editor configuration.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CODE_OF_CONDUCT.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+## Carbon Code of Conduct
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CODE_OF_CONDUCT.md:1]: Should be at root of dev-patterns.

Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CODE_OF_CONDUCT.md:1]: just 
## Code of Conduct

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CODE_OF_CONDUCT.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+If you are subjected to or witness unacceptable behavior, or have any other concerns, please communicate with us directly via carbon@us.ibm.com.
+
+### Slack
+
+1. Please, no `@here` or `@channel` unless it is a bug that is **breaking production code.** We have communities in many different time zones, and we would hate to disrupt someone not at the office.
+2. There's no need to notify the Carbon team that you have made an issue in Carbon's GitHub repositories. We check these several times a day, so we promise we'll address your issue as soon as we can.
+3. Reserve direct messages for sensitive discussions, such as privacy, legal, or HR reasons. Otherwise, we encourage posting in public channels so people with the same questions can benefit.
+4. If you have a pressing bug fix, it is best to make a PR directly to get your issues addressed.
+5. `#carbon-components`: This channel is for questions about Carbon Components only. It is not the place to ask general coding questions. Instead, use a dev community like [StackOverflow](https://stackoverflow.com/).
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CODE_OF_CONDUCT.md:27]: dev-patterns specific

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
+# Contribution Guidelines
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/CONTRIBUTING.md:1]: into the root dev-patterns, and dev-patterns specific.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/ISSUE_TEMPLATE/bug_report.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/ISSUE_TEMPLATE/bug_report.md:1]: dev-patterns specific into dev-patterns root.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/ISSUE_TEMPLATE/feature_request.md [hunk 13]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/ISSUE_TEMPLATE/feature_request.md:1]: dev-patterns specify into dev-patterns root.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/PULL_REQUEST_TEMPLATE.md [hunk 14]
```diff
@@ -0,0 +1,17 @@
+Closes #
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/PULL_REQUEST_TEMPLATE.md:1]: dev-patterns root.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/SUPPORT.md [hunk 15]
```diff
@@ -0,0 +1,7 @@
+# Support
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.github/SUPPORT.md:1]: dev-patterns root.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.npmrc [hunk 16]
```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.npmrc:1]: not needed.

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.prettierignore [hunk 17]
```diff
@@ -0,0 +1,6 @@
+node_modules
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.prettierignore:1]: not needed

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json [hunk 18]
```diff
@@ -0,0 +1,84 @@
+{
+  "name": "ibm-mq-demo",
+  "version": "0.0.0",
+  "private": true,
+  "repository": "git@github.com:carbon-design-system/carbon-tutorial.git",
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json:5]: correct repository

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/public/index.html [hunk 19]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/public/index.html:1]: Need Copyright

File: ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/src/components/MQPatterns/RequestResponse/nodes.js [hunk 20]
```diff
@@ -0,0 +1,143 @@
...
+      role: 'q',
+      depth: 0,
+      queueName: 'DEV.QUEUE.3',
+    },
+    position: { x: 650, y: 20 },
+    sourcePosition: 'right',
+    targetPosition: 'left',
+    draggable: true,    
+  },
+  // {
```
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/src/components/MQPatterns/RequestResponse/nodes.js:60]: These ids will clash with the other pattern. Suggest you prefix with the pattern eg. "RR01" etc.

**Review by chughts — CHANGES_REQUESTED** [review]
see comments above.

File: ibm-messaging-mq-cloud-showcase-app/LICENSE (hunk 2 above)
Comment (chughts) [inline ibm-messaging-mq-cloud-showcase-app/LICENSE]: This is a duplicate of the LICENSE at the root of dev-patterns. This one can be removed.

**Review by chughts — APPROVED** [review]

**Review by rcoppen — APPROVED** [review]

**Pull request merged**